app/views/api-keys/_keys.njk

```diff
@@ -15,7 +15,10 @@
       govukButton({
         text: "Create a new API key",
         href: routes.apiKeys.create,
-        classes: "generate-key"
+        classes: "generate-key",
+        attributes: {
+          id: "create-api-key"
```

Comment:
adding `id`s to elements looks fine with me, in the future when thinking about automated tests we should consider using `data-action="create"` more -- this is a valid HTML5 meta tag that describes what will happen but doesn't impact the actual structure of the DOM. 

In this was the test can look something like
`buttonAction('create').click()`

which will scale no matter how many buttons we have, as long as they all do a unique type of action